openshift-examples/keycloak-https.json

```diff
@@ -42,6 +43,41 @@
             "value": "H2",
             "required": true
         },
+        {
+            "displayName": "DB ADDR",
+            "description": "Specify the name of the database pod if you are going to use an external DB. The name of the DB pod should not be 'mysql' due to an keycloak internal bug",
+            "name": "DB_ADDR",
+            "from": "maximum length of 253 lower case alphanumeric characters, -, and .",
+            "value": "",
+            "required": false
+        },
+        {
+            "displayName": "DB Port",
+            "description": "DB port (default ports for H2 = 9123, MYSQL and MARIADB = 3306, POSTGRES = 5432",
+            "name": "DB_PORT",
+            "value": "9123",
+            "required": true
+        },
+        {
+            "displayName": "DB Database",
+            "description": "Name of the DB scheme",
+            "name": "DB_DATABASE",
+            "value": "keycloak",
+            "required": true
+        },
+        {
+            "displayName": "JDBC Parameters",
+            "description": "additional database driver parameters, if you use 'useSSL=true', you manually need to add a certificate",
+            "name": "JDBC_PARAMS",
+            "value": "useSSL=false",
```

Comment:
Do not set a value for JDBC_PARAMS. useSSL=false is not supported by all DBs

Reply:
than maybe make it an optional empty parameter. I just tested for h2 and mysql